app/code/Magento/InventoryInStorePickupApi/Model/SearchRequestBuilderInterface.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupApi\Model;
```

Comment:
I believe it should be moved to `Api`

Reply:
I would like to have in `Api` folder only things related to webapi.
Moreover, in the `Framework` most of builders not even an `@api`.